### ISSUE TITLE
Receiving message leads to infinite loop of delivery attempts

### ISSUE THREAD

**Issue by andreas-42** [davidgfnet/whatsapp-purple#382]
With the current devel version 7621111:

A contact and I are both logged in, he tries to send me a message. Nothing appears in pidgin, but the log files show delivery attempts about once per second, always failing with the message "Axolotl exception (parseWhisperMessage): No such signedprekeyrecord! 30479 Unknown error".

This goes on forever, until I stop the program, replace libwhatsapp.so with the testing version add0121, and start pidgin again. The old message is then delivered immediately and correctly on login.

@davidgfnet, I'll send you the trace by e-mail.

I'm back to the testing version now, so far I haven't had this problem there...

**Comment by davidgfnet** [comment]
Yep, happened to me as well... I'll try to make use of the "count" counter and see what happens :D

**Comment by davidgfnet** [comment]
Could you check out the devel branch and report back? Thanks!

**Comment by andreas-42** [comment]
OK, running the new devel version with debug enabled now. I'll let you know what happens. ;)

**Comment by davidgfnet** [comment]
Any updates?

**Comment by andreas-42** [comment]
I haven't had any infinite loops after your update.

**Comment by KottV** [comment]
I have infinite receiving with -devel build
